plugins/scaffolder/src/components/fields/EntityTagsPicker/EntityTagsPicker.tsx

```diff
@@ -24,11 +24,21 @@ import { FormControl, TextField } from '@material-ui/core';
 import { Autocomplete } from '@material-ui/lab';
 import { FieldExtensionComponentProps } from '../../../extensions';
 
+/**
+ * The input props that can be specified under `ui:options` for the
+ * `EntityTagsPicker` field extension.
+ *
+ * @public
+ */
 export interface EntityTagsPickerUiOptions {
   kinds?: string[];
 }
+
 /**
- * EntityTagsPicker
+ * The underling component that is rendered in the form for the `EntityTagsPicker`
```

Comment:
```suggestion
 * The underlying component that is rendered in the form for the `EntityTagsPicker`
```

Signed-off-by: Fredrik Adelöw <freben@gmail.com>